src/BeliefTypes.jl

```diff
@@ -66,16 +66,24 @@ end
 TreeBelief(p::BallTreeDensity, inferdim::Real=0.0) = TreeBelief(getPoints(p), getBW(p), inferdim)
 TreeBelief(val::Array{Float64,2}, bw::Array{Float64,2}, inferdim::Real=0.0) = TreeBelief(val, bw, inferdim)
 
+"""
+    CliqStatus
+Clique status message enumerated type with status:  
+initialized, upsolved, marginalized, downsolved, uprecycled
+"""
+@enum CliqStatus initialized upsolved marginalized downsolved uprecycled
```

Comment:
oh cool, didn't know you can do that.  There is a small chance that these are Symbol under the hood, but either way this is a much cleaner representation.

Reply:
It defaults to Int32 see comment : https://github.com/JuliaRobotics/IncrementalInference.jl/pull/442#discussion_r353575325